plugin/src/main/java/io/jenkins/plugins/casc/SecretSourceResolver.java

```diff
@@ -107,10 +108,7 @@ public String lookup(String key) {
         static final FileStringLookup INSTANCE = new FileStringLookup();
 
         @Override
-        public String lookup(final String key) {
-            if (key == null) {
-                return null;
-            }
+        public String lookup(@NonNull final String key) {
```

Comment:
[Can never be null](https://github.com/apache/commons-text/blob/842a0880e524a9895b87af320c95ad144d43b1b9/src/main/java/org/apache/commons/text/lookup/InterpolatorStringLookup.java#L141-L143)